qa/standalone/scrub/osd-scrub-test.sh

```diff
@@ -555,10 +555,10 @@ function TEST_dump_scrub_schedule() {
     # a time in the future:
     # e.g. 'periodic scrub scheduled @ 2021-10-12T20:32:43.645168+0000'
 
-    declare -A expct_starting=( ['query_active']="false" ['query_is_future']="true" ['query_schedule']="scrub scheduled" )
+    declare -A expect_starting=( ['query_active']="false" ['query_is_future']="true" ['query_schedule']="scrub scheduled" )
     declare -A sched_data
     extract_published_sch $pgid $now_is "2019-10-12T20:32:43.645168+0000" sched_data
-    schedule_against_expected sched_data expct_starting "initial"
+    schedule_against_expected sched_data expect_starting "initial"
```

Comment:
Leave this alone, shortening in code is not uncommon, we gain nothing by changing it.